### ISSUE TITLE
Can you please remove this account?

### ISSUE THREAD

**Issue by h6w** [phpvirtualbox-zz/phpvirtualbox#1]
I want to create an "organisation" with the name "phpvirtualbox" so that we can transfer the existing codebase into it and manage it as a group.

However, it seems you have created both a user and an empty repository under that name, neither of which has been used in 18 months.

Unfortunately, Github doesn't let me create an organisation if a user with the same name exists already.

Considering this account is empty, would you mind completely deleting it so we can use the name?

If you are interested in contributing to the phpvirtualbox project, you would be more than welcome to join the new organisation when I'm able to create it.

Thanks!

Cheers,
Tudor.

**Comment by phpvirtualbox-zz** [comment]
Sure I'll do that tomorrow

**Comment by h6w** [comment]
Awesome.  Thanks! :-)

**Comment by h6w** [comment]
Bump.  How's that going? :-)

**Comment by ghost** [comment]
Any infos on that?

**Comment by h6w** [comment]
Bumping again.  Can we please do this ASAP?  Thanks!

**Comment by imoore76** [comment]
It turns out I don't own the phpvirtualbox account. It may be a squatter. Someone also picked up phpvirtualbox.org and offered to sell it to me

**Comment by h6w** [comment]
Hallo @imoore76!   Yes.  It's not you.  I've messaged that person and they agreed to close the account, but so far it hasn't happened.  I have messaged github to see if they can take that as proof of intent and convert it to an organisation for me.  I'll keep everyone posted.

**Comment by imoore76** [comment]
Ive transferred the repo. Thanks for taking over. I really don't have the time anymore

**Comment by h6w** [comment]
@imoore76 No worries.  I'll leave you as one of the owners of the organisation in case I fall under a bus someday and the project needs rescuing. :-)  Thanks so much for all your work.

Is it possible you can send the SF details to tudor@tudorholton.com?  Once I've migrated the questions to GitHub, I want to point the issues page to Github so we don't have duplicate issues being posted in future.

**Comment by DerMozart** [comment]
So where is the new repository now? ist it phpvirtualbox/phpvirtualbox? There is an open pull req for 5.1. Anyone knows what's happening there?

**Comment by h6w** [comment]
@DerMozart Yes.  The correct repository is phpvirtualbox/phpvirtualbox.  I've been looking at the pull request.
